plugins/telemetry/config.go

```diff
@@ -36,26 +36,26 @@ type Config struct {
 	Skipped []string `json:"skipped"`
 }
 
-func defaultConfig() *Config {
+var DefaultConfig = func() *Config {
 	return &Config{
 		PollingInterval: defaultUpdatePeriod,
 	}
 }
 
 // loadConfig returns telemetry plugin file configuration if exists
 func (p *Plugin) loadConfig() (*Config, error) {
-	cfg := defaultConfig()
+	cfg := DefaultConfig()
```

Comment:
Just to be sure, add check if the `DefaultConfig` is nil. If it is, use zero value of `Config`.